spec/decorators/odk/form_decorator_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+# Form decorator spec
+describe Odk::FormDecorator, :odk, :reset_factory_sequences do
+  include_context "odk rendering"
+
+  describe "needs_manifest?" do
+    let(:decorated_form) { decorate(form) }
+
+    context "for form with single level option sets only" do
+      let(:form) { create(:form, question_types: %w[select_one]) }
+      it "should return false" do
+        expect(decorated_form.needs_manifest?).to be_falsey
+      end
+    end
+
+    context "for form with multi level option set" do
+      let(:form) { create(:form, question_types: %w[select_one multilevel_select_one]) }
+      it "should return true" do
+        expect(decorated_form.needs_manifest?).to be_truthy
```

Comment:
Question: what is the difference between `be true` and `be_truthy`? Given that, which is more appropriate here? Why?

Reply:
if the expectation expression is returning a boolean, you can append an underscore to `be` and have things like be_truthy, be_falsey, be_valid, be_any_boolean_return_value

after doing some googling, it's said that be_truthy passes when the expectation is neither nil nor false while be true uses a `==` to do its comparison

Is there a preference for the project?